src/runtime/auto-prompt-manager.ts

```diff
@@ -550,21 +552,39 @@ export class AutoPromptManager {
     autoPromptType?: AutoPromptType;
     isClosable?: boolean;
   }): () => void {
-    return () => {
-      const params: AudienceActionIframeParams = {
-        action,
-        configurationId,
-        autoPromptType,
-        onCancel: () => this.storeLastDismissal_(),
-        isClosable,
+    if (action == TYPE_REWARDED_ADS) {
```

Comment:
nit: in context, it appears `actionType` or similar would be a clearer name for this variable.

Reply:
Done.